src/content/en/tools/lighthouse/audits/contrast-ratio.md

```diff
@@ -0,0 +1,57 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Background and Foreground Colors Have Sufficient Contrast Ratio" Lighthouse audit.
+
+{# wf_updated_on: 2017-01-20 #}
+{# wf_published_on: 2017-01-20 #}
+
+# Background and Foreground Colors Have Sufficient Contrast Ratio  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Some users have low-contrast vision. They don't see many areas that are very
+bright or very dark. Everything looks about the same brightness. If the text
+doesn't have a high contrast ratio, these users literally have trouble seeing
+it at all.
+
+## How to pass the audit {: #how }
+
+To find and fix each of element that does not have a sufficient contrast ratio:
+
+1. Install [ChromeLens][CL].
```

Comment:
@alice what I'm trying to solve here is to give users a relatively easy path for finding the elements that are failing the test. The Lighthouse extension just gives you a list of selectors like `.nav > .button`. I already mention that you can use the DevTools Console to query for the elements, but that can get kind of tedious. So I was just providing another path for finding the elements, hence ChromeLens, which highlights the failing elements in your viewport...